client/state/reader/start/test/actions.js

```diff
@@ -80,7 +84,37 @@ describe( 'actions', () => {
 
 				expect( dispatchSpy ).to.have.been.calledWith( {
 					type: READER_SITE_UPDATE,
-					payload: [ sampleSuccessResponse.recommendations[0].meta.data.site ]
+					payload: [ sampleSuccessResponse.recommendations[ 0 ].meta.data.site ]
+				} );
+			} ).catch( ( err ) => {
+				assert.fail( err, undefined, 'errback should not have been called' );
+			} );
+		} );
+	} );
+
+	describe( '#requestGraduate', () => {
+		const sampleResponse = { success: true };
+		useNock( nock => {
+			nock( 'https://public-api.wordpress.com:443' )
+				.post( '/rest/v1.1/read/graduate-new-reader' )
```

Comment:
I wondered that! ...but if you specify v1.2 in the request URL here, nock throws an error.

The endpoint works with either version.